### ISSUE TITLE
Saving PDF to another directory

### ISSUE THREAD

**Issue by rickmedlin** [eKoopmans/html2pdf.js#556]
This is a very easy to use and robust library.  Thanks for making it, as it satisfied my requirements almost exactly.

Is it possible to save a PDF to somewhere other than the default download folder, for example:

html2pdf().from(invoice).set(opt).save("D:\Reports\ReportFolder");

If not, is this feature under consideration? I underxtand that there might be security issues, but it would be nice to have.

**Comment by RobbyRabbitman** [comment]
browser permit this, no?

**Comment by rickbps** [comment]
True. I had decided to use the default Downloads folder, but I'd like to send the generated PDF as an e-mail attachment.

Is there a way to render it on the server side so as to avoid uploading it just so it can be attached?

**Comment by joem824** [comment]
@eKoopmans will there be a future updates for regarding the saving to another directory?
